tests/phpunit/integration/tests/Integrations/Jetpack.php

```diff
@@ -245,8 +244,6 @@ public function test_format_milliseconds( $milliseconds, string $string ): void
 
 	/**
 	 * @param mixed $value
-	 * @param int $object_id
-	 * @param string $meta_key
```

Comment:
Why remove this?

Reply:
Because the linter/fixer removes unnecessary PHPDoc blocks that are already documented with the type hints.

So in the function signature there's `int $object_id`, so adding `@param int $object_id` PHPDoc is redundant (unless there is an actual description). So `@param int $object_id Object ID.` would be kept because it provides additional information.